.docker/readme.md

```diff
@@ -36,6 +36,13 @@ Follow these steps to setup a local WordCamp.org environment using [Docker](http
 	composer install
 	```
 
+1. Install 3rd-party JS packages and build the CSS & JS needed for some projects. You'll need [node](https://nodejs.org/) & [yarn](https://yarnpkg.com/). Optionally you can use [nvm](https://github.com/nvm-sh/nvm) to keep your node version up to date. Run the following to install and build each project (omit `nvm` command if you're not using it).
```

Comment:
```suggestion
1. Install 3rd-party JS packages and build the CSS & JS needed for some projects. You'll need [node](https://nodejs.org/) & [yarn](https://yarnpkg.com/). Optionally you can use [nvm](https://github.com/nvm-sh/nvm) to keep your node version up to date. Running the following will install and build all of the projects in one step (omit `nvm` command if you're not using it).
```

When I first read this I thought it meant I would need to run these commands for each project. So this is just a minor wording change that I think makes it clearer, but you can take it or leave it.

Reply:
Hm, I see what you mean— the yarn workspaces run build command does actually run through each project (workspace) itself, which is why I said it like that, but this is more clear to folks who don't know that 👍🏻